ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -282,6 +291,16 @@ internal class CustomerCenterViewModelImpl(
         context.startActivity(Intent.createChooser(intent, "Contact Support"))
     }
 
+    @SuppressWarnings("ForbiddenComment")
+    override fun openURL(context: Context, url: Uri) {
+        // TODO: Handle In-App Browser
+        try {
+            context.startActivity(Intent(Intent.ACTION_VIEW, url))
+        } catch (e: ActivityNotFoundException) {
+            Logger.e("Error opening URL", e)
```

Comment:
Hmm I wonder if we should display a Toast or something like that in this case, so the customer knows something is happening... We already do that in the paywall footer: https://github.com/RevenueCat/purchases-android/blob/main/ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt#L264

We might want to extract that logic and copy it here.

Reply:
updated!